### PR TITLE
Add CustomGlobalExceptionHandler and data validation.

### PR DESCRIPTION
**Pull request by makaroshyna**

### CODE REVIEW

**Review by JohnGrey17 — APPROVED** [review]
GJ

File: src/main/java/makaroshyna/onlinebookstore/dto/CreateBookRequestDto.java [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
 
 @Data
 public class CreateBookRequestDto {
+    @NotBlank
     private String title;
+    @NotBlank
     private String author;
+    @Size(min = 13)
+    @NotBlank
     private String isbn;
```
Comment (andrii-hoienko) [inline src/main/java/makaroshyna/onlinebookstore/dto/CreateBookRequestDto.java]: There is specific annotation to validate isbn

File: src/main/java/makaroshyna/onlinebookstore/exception/CustomGlobalExceptionHandler.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    private String getErrorMessage(ObjectError error) {
+        if (error instanceof FieldError fieldError) {
+            String fieldName = fieldError.getField();
+            String errorMessage = fieldError.getDefaultMessage();
+            return fieldName + " " + errorMessage;
+        }
+        return error.getDefaultMessage();
+    }
```
Comment (andrii-hoienko) [inline src/main/java/makaroshyna/onlinebookstore/exception/CustomGlobalExceptionHandler.java]: Please add methods to handle exceptions that could occur in your code

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
Good job, but some improvements required

**Pull request merged**